### PR TITLE
feat(metrics): add metrics for a few automations

### PR DESCRIPTION
**Pull request by bcoe**
Retrofit a few of our bots with the new `logger.metric` functionality, this data now streams directly into BigQuery for dashboard goodness.

Refs: #1655

### CODE REVIEW

File: packages/do-not-merge/src/do-not-merge.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export = (app: Probot) => {
         }
         return;
       }
-      logger.info('Do not merge label found');
+      logger.metric('do_not_merge.add_label');
```
Comment (tbpg) [inline packages/do-not-merge/src/do-not-merge.ts:81]: Might want to move this down, right next to the `Creating failed check` log. That way, each PR is only counted once.

**Review by tbpg — CHANGES_REQUESTED** [review]

File: packages/release-please/src/release-please.ts [hunk 2]
```diff
@@ -383,4 +383,21 @@ export = (app: Probot) => {
...
   });
+
+  app.on('release.created', async context => {
+    const repoUrl = context.payload.repository.full_name;
+    const remoteConfiguration = (await context.config(
+      WELL_KNOWN_CONFIGURATION_FILE
+    )) as ConfigurationOptions | null;
+
+    // If no configuration is specified,
+    if (!remoteConfiguration) {
```
Comment (bcoe) [inline packages/release-please/src/release-please.ts:394]: `releae-please` was already subscribing to `release.created` events.

File: packages/do-not-merge/src/do-not-merge.ts (hunk 1 above)
Comment (bcoe) [reply packages/do-not-merge/src/do-not-merge.ts]: done 👍

**Review by sofisl — APPROVED** [review]

**Review by tbpg — APPROVED** [review]
Thanks!

**Review by danielbankhead — APPROVED** [review]

**Pull request merged**